conversion.qmd

```diff
@@ -156,4 +156,29 @@ A slice is a representation of a sequence of elements that are part of a larger
 
 The R runtime performs [string interning](https://en.wikipedia.org/wiki/String_interning) to
 all of its string elements. This means, that whenever R encounters a new string,
-it adds it to its internal string intern pool. 
\ No newline at end of file
+it adds it to its internal string intern pool. Let's look at a concrete example:
```

Comment:
what is a "string intern pool"

Can we add a callout-tip here?

Reply:
Done